contracts/DebtToken.sol

```diff
@@ -146,17 +146,7 @@ contract DebtToken is MintableNonFungibleToken, Pausable {
         if (registry.getBeneficiary(bytes32(_tokenId)) != _to) {
             registry.modifyBeneficiary(bytes32(_tokenId), _to);
         }
-    }
 
-    /**
-     * We oveerride the core ownership getter of the parent non-fungible token
-     * contract so that it retrieves the debt's current beneficiary from the debt registry
-     */
-    function _ownerOf(uint _tokenId)
-        internal
-        view
-        returns (address _owner)
-    {
-        return registry.getBeneficiary(bytes32(_tokenId));
+        super._setTokenOwner(_tokenId, _to);
```

Comment:
Should this also not be in the if block? Or do we always want to call `super`?

Reply:
Good catch -- changing now.

Reply:
Actually -- I dug a little deeper and we definitely need to be calling `super` every time.  If we don't override `ownerOf` in addition to `setTokenOwner`, then we're essentially maintaining parallel redundant stores of token ownership in the `DebtToken` and `DebtRegistry` -- in which case, our tests break if we're not consistently calling `super`.  LMK if you have any questions